iroh-ctl/src/fixture.rs

```diff
@@ -0,0 +1,42 @@
+use std::collections::HashMap;
+use std::env;
+
+use iroh::{MockApi, MockP2p};
+use libp2p::PeerId;
+
+type GetFixture = fn() -> MockApi;
+type FixtureRegistry = HashMap<String, GetFixture>;
+
+fn fixture_peer_ids() -> MockApi {
+    let mut api = MockApi::default();
+    api.expect_p2p().returning(|| {
+        let mut mock_p2p = MockP2p::default();
+
+        mock_p2p.expect_peer_ids().returning(|| {
+            let peer_id0 = "1AXRDqR8jTkwzGqyu3qknicAC5X578zTMxhAi2brppK2bB"
+                .parse::<PeerId>()
+                .unwrap();
+            let peer_id1 = "1Ag5LvC3vwsQicTy1dxNdd1xiNxLUS4Aic4NtdHBqAHD3j"
+                .parse::<PeerId>()
+                .unwrap();
+            Ok(vec![peer_id0, peer_id1])
+        });
+        Ok(mock_p2p)
+    });
+    api
+}
+
+fn register_fixtures() -> FixtureRegistry {
+    let mut registry = FixtureRegistry::new();
+    registry.insert("peer_ids".to_string(), fixture_peer_ids);
+    registry
```

Comment:
fyi you can also write

```rust
[("peer_ids".to_string(), fixture_peer_ids)].into_iter().collect()
```

to populate a hashmap

Reply:
Unfortunately I'm getting

```
32   |         .collect()
     |          ^^^^^^^ value of type `HashMap<std::string::String, fn() -> MockApi>` cannot be built from `std::iter::Iterator<Item=(std::string::String, fn() -> MockApi {fixture_peer_ids})>`
```
I had to add a `as GetFixture` to fix that.